### ISSUE TITLE
Update Error Dialog- pgmodeler-0.9.0-beta

### ISSUE THREAD

**Issue by dudisbr** [pgmodeler/pgmodeler#940]
Everytime pgModeler tries to check for updates an error dialog is displayed.
![captura de tela 2017-04-11 as 22 53 07](https://cloud.githubusercontent.com/assets/25533768/24937881/1c311176-1f0a-11e7-80f2-242a6f87ae9a.png)

**Comment by GeertJohan** [comment]
It seems the website is down; no A record for that domain name.

**Comment by rkhaotix** [comment]
Hey guys! Sorry the delay on response!

The problem is that I've switched the project's site from http to https and update mechanism in pgModeler is still referencing the http address. I've fixed that in the beta1 (already in development). IBut, I'm still trying to figure out a solution for that on prior releases.

**Comment by GeertJohan** [comment]
Okay, but the A record is also missing...
The error dialog is quite anoying because it keeps poping up when you're working.
```
$ dig www.pgmodeler.com.br

; <<>> DiG 9.10.3-P4-Ubuntu <<>> www.pgmodeler.com.br
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 14334
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;www.pgmodeler.com.br.		IN	A

;; Query time: 8 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Thu Apr 13 13:43:39 CEST 2017
;; MSG SIZE  rcvd: 49
```

**Comment by GeertJohan** [comment]
You probably have a forwarder on :80 (http), perhaps you can setup a exception route (the update checker path) that doesn't forward but proxies, so that it's still served on http.

**Comment by rkhaotix** [comment]
About the error dialog, you can disable it in General settings. 
I'm not a network expert but I'll see what I can do.

**Comment by rkhaotix** [comment]
Oh, the site is down too... :(

**Comment by GeertJohan** [comment]
Yes, the missing DNS record is unforgiving to both updaters and browsers :)

**Comment by JedrzejczykRobert** [comment]
Site working OK, but error message stil exists

**Comment by JedrzejczykRobert** [comment]
The problem is when website serwer redirect connection from http to https. 

The temporary resolution is manual patch file utils.dll:

"http://www.pgmodeler.com.br" => "https://www.pgmodeler.com.br"
"https://github.com/pgmodeler/pgmodeler/issues" => "http://github.com/pgmodeler/pgmodeler/issues"

You can download patched files from:

https://prog.olsztyn.pl/test/pgmodeler/0.9.0-alpha1-pached-utils.dll.zip
https://prog.olsztyn.pl/test/pgmodeler/0.9.0-beta-pached-utils.dll.zip

**Comment by dudisbr** [comment]
I'm using pgmodeler on Mac.

**Comment by JedrzejczykRobert** [comment]
libutils/src/globalattributes.cpp line 30

PGMODELER_SITE=QString("https://www.pgmodeler.com.br"),

**Comment by dudisbr** [comment]
I think this problem was solved on PgModeler 0.9.1-alpha.
